InvoiceMaker/Controllers/FeeLineItemController.cs

```diff
@@ -0,0 +1,58 @@
+﻿using InvoiceMaker.Data;
+using InvoiceMaker.FormModels;
+using InvoiceMaker.Models;
+using InvoiceMaker.Repositories;
+using System;
+using System.Collections.Generic;
+using System.Data.Entity.Infrastructure;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+
+namespace InvoiceMaker.Controllers
+{
+    public class FeeLineItemController : Controller
+    {
+        private Context context;
+
+        public FeeLineItemController()
+        {
+            context = new Context();
+        }
+        // GET: FeeLineItems
+        public ActionResult Index()
+        {
+            return View();
+        }
+
+        [HttpGet]
+        public ActionResult Create(int id)
+        {
+            CreateFeeLineItem feeLineItem = new CreateFeeLineItem();
+            return View("Create", feeLineItem);
+
+        }
+
+        [HttpPost]
+        public ActionResult Create(int id, CreateFeeLineItem feeLineItem)
+        {
+            FeeLineItemRepository repo = new FeeLineItemRepository(context);
+            try
+            {
+                DateTimeOffset when = DateTimeOffset.Now;
+                FeeLineItem newFeeLineItem = new FeeLineItem(0, feeLineItem.Amount, feeLineItem.Description, when);
+                newFeeLineItem.InvoiceId = id;
+                InvoiceRepository invRepo = new InvoiceRepository(context);
+                var invoice = invRepo.GetInvoice(id);
+                invoice.FeeLineItems.Add(newFeeLineItem);
```

Comment:
Setting the `newFeeLineItem.InvoiceId` foreign key property should be enough to create the relationship between this fee line item and the invoice. Did you test without adding the fee line item to the invoice's `FeeLineItems` collection?